### ISSUE TITLE
osu!direct doesn't filter out recently downloaded maps.

### ISSUE THREAD

**Issue by mm201** [ppy/osu-stable-issues#937]
1. Ensure "Show already downloaded maps" is unchecked.
2. Do an osu!direct search. For instance, a mapper's name.
3. Download some maps.
4. Do a different but similar search. For instance, the name of a mapper who often guests for the first mapper.
5. Observe that some songs you just downloaded are showing up.

You have to go into Song Select and make it refresh, then back to osu!direct, to make them stay away.

**Comment by peppy** [comment]
Makes sense as to why this would be the case, but won't be changing on stable. In lazer this kind of thing updates immediately as one would expect.
